src/init.js

```diff
@@ -84,3 +85,44 @@ EventBus.$on('signaling-recording-status-changed', (token, status) => {
 		showError(t('spreed', 'The recording failed. Please contact your administrator.'))
 	}
 })
+
+/**
+ * Migrate localStorage to @nextcloud/browser-storage
+ *
+ * In order to preserve the user settings while migrating to the abstraction,
+ * we loop over the localStorage entries and add the matching ones to the
+ * BrowserStorage
+ */
+const migrateDirectLocalStorageToNextcloudBrowserStorage = () => {
+	const storageKeys = Array.from(Array(localStorage.length), (i) => localStorage.key(i)).filter(key => key.startsWith('audioDisabled_')
```

Comment:
**issue (blocking):** Sorry, I made a typo on the last suggestion, `i` must be index, not item.

```suggestion
	const storageKeys = Array.from(Array(localStorage.length), (_, i) => localStorage.key(i)).filter(key => key.startsWith('audioDisabled_')
```

Or, if `Array.from` looks complex:

```suggestion
	const storageKeys = Array(localStorage.length).fill().map((_, i) => localStorage.key(i)).filter(key => key.startsWith('audioDisabled_')
```